src/POData/UriProcessor/UriProcessor.php

```diff
@@ -152,21 +152,57 @@ public function execute()
         $requestMethod = $operationContext->incomingRequest()->getMethod();
         if ($requestMethod == HTTPRequestMethod::GET()) {
             $this->executeGet();
+        } elseif ($requestMethod == HTTPRequestMethod::POST()) {
+            $this->executePost();
         } elseif ($requestMethod == HTTPRequestMethod::PUT()) {
             $this->executePut();
+        } elseif ($requestMethod == HTTPRequestMethod::DELETE()) {
+            $this->executeDelete();
+            //TODO: we probably need these verbes eventually.
+        /*} elseif ($requestMethod == HTTPRequestMethod::PATCH()) {
+            $this->executePatch();
+        } elseif ($requestMethod == HTTPRequestMethod::MERGE()) {
+            $this->executeMerge();*/
         } else {
             throw ODataException::createNotImplementedError(Messages::onlyReadSupport($requestMethod));
         }
     }
 
+ 
+
```

Comment:
For grokkability mainly - can you make the inter-method vertical spacing consistent?  eg N blank lines (where N might be 1), then phpdoc, then method sig?